packages/arb-token-bridge-ui/src/components/common/SearchPanel/SearchPanelTable.tsx

```diff
@@ -2,20 +2,20 @@ import { MagnifyingGlassIcon } from '@heroicons/react/24/outline'
 import React, { PropsWithChildren } from 'react'
 
 type SearchPanelTableProps = {
-  searchInputPlaceholder: string
-  searchInputValue: string
+  searchInput: {
+    placeholder: string
+    value: string
+    onChange: React.ChangeEventHandler<HTMLInputElement>
+  }
```

Comment:
Passing an object here will either force us to wrap it with `useCallback`, or to rerender all the time

Reply:
ok good point, i'll update